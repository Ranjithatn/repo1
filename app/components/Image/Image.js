import React, { Fragment } from "react";

const Image = ({ className, id, alt, src, width, height, onClick, title, style={} }) => {
  return (
    <img
      onClick={onClick}
      id={id}
      title={title}
      className={"img" + (className !== undefined ? " " + className : "")}
      src={src}
      alt={alt}
      width={width}
      height={height}
      style={style}
    />
  );
};

export default Image;
