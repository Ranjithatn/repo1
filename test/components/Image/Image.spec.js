import React from "react";
import toJson from "enzyme-to-json";
import Image from "../../../app/components/Image/Image";
import Adapter from "enzyme-adapter-react-16";
import { configure, shallow, mount } from "enzyme";


configure({ adapter: new Adapter() });


describe('<Image />', () => {

  it("should render without crashing", () => {
    const component = mount(
      <Image />
    );
    const tree = toJson(component);
    expect(tree).toMatchSnapshot();
  });
});
