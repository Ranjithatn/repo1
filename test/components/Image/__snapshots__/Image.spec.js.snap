// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Image /> should render without crashing 1`] = `
<Image>
  <img
    className="img"
    style={Object {}}
  />
</Image>
`;
